rootfs/templates/scaffolding/Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV KOPS_STATE_STORE_REGION "{{ getenv "KOPS_STATE_STORE_REGION" "us-east-1" }}"
 
 # https://github.com/kubernetes/kops/blob/master/channels/stable
 # https://github.com/kubernetes/kops/blob/master/docs/images.md
-ENV KOPS_BASE_IMAGE="{{ getenv "KOPS_BASE_IMAGE" "kope.io/k8s-1.7-debian-jessie-amd64-hvm-ebs-2017-07-28" }}"
+ENV KOPS_BASE_IMAGE="{{ getenv "KOPS_BASE_IMAGE" "kope.io/k8s-1.10-debian-jessie-amd64-hvm-ebs-2018-08-17" }}"
```

Comment:
Use 
`kope.io/k8s-1.11-debian-stretch-amd64-hvm-ebs-2018-08-17`
as this is recommended version for k8s `1.10.11` 
Check here https://github.com/kubernetes/kops/blob/master/channels/stabl

Reply:
@goruha That link isn't working. As mentioned in the commit I grabbed this from https://github.com/kubernetes/kops/blob/master/channels/stable#L22-L24